### ISSUE TITLE
Make 'GRC' more configurable (multi-configuration).

### ISSUE THREAD

**Issue by tmicka23** [arminbro/generate-react-cli#10]
**Is your feature request related to a problem? Please describe.**

Hi, thank you for this superb and very practical library.
I would like to suggest an improvement.
currently in version 4.0.2, I can do `generate-react component <MyComponent>`
which makes `MyApp / src / components / <Mycomponent> / <Mycomponent>.js + <Mycomponent>.css + <Mycomponent>.test.js`

**Describe the solution you'd like**

I would like to have the possibility of creating a 2nd (or more) configuration like `generate-react page Home` which does` MyApp / src / pages / Home / Home.js + Home.css + Home.test.js`

**Describe alternatives you've considered**

see this example configuration file
```json
{
  "component": {
    "path": "src/components",
    "css": {
      "preprocessor": "css",
      "module": true,
      "withStyle": true
    },
    "test": {
      "library": "None",
      "withTest": true
    },
    "withStory": false,
    "withLazy": false
  },
  "page": {
    "path": "src/pages",
    "css": {
      "preprocessor": "css",
      "module": true,
      "withStyle": true
    },
    "test": {
      "library": "None",
      "withTest": true
    },
    "withStory": false,
    "withLazy": false
  },
  "usesTypeScript": false
}
```

Thanks for read my issue. thanks for this project !

**Comment by arminbro** [comment]
@tmicka23 thanks for the suggestion! The funny thing is that my buddy suggested the same thing like a week ago. 😄

I think this feature request makes sense and will improve on generating pages with their own default configurations in place.

The plan is to add a new `page` command that will have its own configuration rules that you can set up just like you did for the `component` command. You will also be able to pass the same options to the `page` command that you are already familiar with in the `component` command.

Once this new feature is published, the next time you run the new version of generate-react-cli, it will ask a few questions so it can update your **generate-react-cli.json** config file with the new `page` configuration rules.

Give me a couple of days and I should have this feature ready and published.

Thanks again for the request.
Armin

**Comment by arminbro** [comment]
@tmicka23 I just released a new version of Generate React CLI ([v4.1.0](https://github.com/arminbro/generate-react-cli/commit/3a441dede662bf6a3d65c67072b50900ece46879)) that now has a new `page` command.

As an example, you should now be able to run `npx generate-react-cli page HomePage` and it will generate a new HomePage component using its own config rules from the **generate-react-cli.json** config file.

If you find something not working correctly or as you expected, please don't hesitate to reach out and file/report a bug so I can take care of it.

Thanks
Armin
